README.md

```diff
@@ -1,2 +1,8 @@
-# maximus
-testing
+# hello-world
+Top 5 Movies
+
+1. Garden State
+2. Superbad
```

Comment:
Can we replace this one with a movie that is not in the comedy category?

Reply:
Sure thing!